src/tests/ftest/nvme/pool_exclude.yaml

```diff
@@ -33,36 +33,42 @@ pool:
   nvme_size: 300000000000
   svcn: 4
   control_method: dmg
-  rebuild_timeout: 120
+  rebuild_timeout: 180
   pool_query_timeout: 30
 container:
   type: POSIX
   control_method: daos
   properties: cksum:crc16,cksum_size:16384,srv_cksum:on,rd_fac:2
-  oclass: RP_3G6
+ior_flags_common: &ior_flags_common
+  write_flags: "-w -F -k -G 1"
+  read_flags: "-F -r -R -k -G 1"
+  api: DFS
+  ior_test_sequence:
+    #   - [scm_size, nvme_size, transfersize, blocksize]
+    #    The values are set to be in the multiples of 10.
+    #    Values are appx GB.
+    - ["NA", "NA", 4000, 50000000]          # [4K, 51M]
+    - ["NA", "NA", 1000000, 500000000]      # [1M, 512M]
+    - ["NA", "NA", 1000000000, 8000000000]  # [1G, 8G]
 ior:
   client_processes:
     np: 48
   test_file: /testFile
   repetitions: 10
-  dfs_destroy: false
-  iorflags:
-    write_flags: "-w -F -k -G 1"
-    read_flags: "-F -r -R -k -G 1"
-    api: DFS
-    dfs_oclass: RP_3G6
-    dfs_dir_oclass: RP_3G6
-    ior_test_sequence:
-      #   - [scm_size, nvme_size, transfersize, blocksize]
-      #    The values are set to be in the multiples of 10.
-      #    Values are appx GB.
-      - [50000000000, 300000000000, 4000, 50000000]          # [4K, 51M]
-      - [50000000000, 300000000000, 1000000, 500000000]      # [1M, 512M]
-      - [50000000000, 300000000000, 1000000000, 8000000000]  # [1G, 8G]
-test_obj_class:
-  oclass:
-    - RP_3G6
-    - RP_4G1
+  dfs_destroy: False
+  iorflags: !mux
+    rp_3g6:
+      <<: *ior_flags_common
+      dfs_oclass: RP_3G6
+      dfs_dir_oclass: RP_3G6
+    ec_4p2gx:
+      <<: *ior_flags_common
+      dfs_oclass: EC_4P2GX
+      dfs_dir_oclass: EC_4P2GX
+    ec_8p2gx:
+      <<: *ior_flags_common
+      dfs_oclass: EC_8P2GX
+      dfs_dir_oclass: EC_8P2GX
```

Comment:
Just FYI that IOR doesn't use `dfs_dir_oclass`. We set this a lot, and there's a default value of `dfs_dir_oclass=SX`, but IOR only works on files, not directories. Okay to keep this for now - eventually I'd like to clean it all up and remove the unused `dfs_dir_oclass` param from the utils.